eng/doc/fips/README.md

```diff
@@ -209,31 +215,30 @@ The following sections describe how to enable FIPS mode and the effect of the `G
 
 To set FIPS mode on Linux, use one of the following options. The first match in this list wins:
 
+- Explicitly enable it by setting the environment variable `GODEBUG=fips140=on`.
 - Explicitly enable it by setting the environment variable `GOFIPS=1`.
 - Explicitly disable it by setting the environment variable `GOFIPS=0`.
 - Implicitly enable it by booting the Linux Kernel in FIPS mode.
   - The Linux Kernel's FIPS mode sets the content of `/proc/sys/crypto/fips_enabled` to `1`. The Go runtime reads this file.
```

Comment:
Maybe I'm just not connecting a few dots, but (related to https://github.com/microsoft/go/pull/1513#issuecomment-2613282156), how do these work?

`GODEBUG=fips140=on OPENSSL_FORCE_FIPS_MODE=1 go test .`
Seems described by this list: turned on `fips140`, and configuring OpenSSL makes it so that when the Go runtime checks whether FIPS mode is enabled, it gets a `true`.

`OPENSSL_FORCE_FIPS_MODE=1 go test .`
Based on this doc, I'm not sure what this will do. From the ADR, I'd expect OpenSSL's mode to be detected and for the runtime to internally set `GODEBUG=fips140=on` (essentially).

(Or, more broadly, if I use `OPENSSL_CONF`.)

Reply:
Hmm, when writing the ADR I was thinking on a Mariner / Azure Linux  booted in FIPS mode or Windows with the FIPS policy enabled, and that's what I end up implementing. But you are right, having the FIPS mode enabled at the OpenSSL config level should also set `GODEBUG=fips140=on`. Will fix that, thanks!

Reply:
https://github.com/microsoft/go/pull/1519